blog/_posts/2016-07-15-team-joda-says-hello.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Team joda says hello
+layout: post
+created_at: Thu Jul 14 2016
+permalink: blog/2016-07-15-team-joda-says-hello
+current: blog
+author: Johanna Lang and Dayana Mick
+twitter: Teamjoda2016
+categories:
+- student-posts
+---
+
+
+
+# “Always two there are, no more, no less.”
+![Team Joda](img/blog/2016/team_joda.jpg)<font color="grey"><small><i>jodas in their coaching company's office (Image: Team joda)</i></small></font>
```

Comment:
the image path is missing a `/` at the beginning (it should be `/img/blog/..`)
can you change that?